<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParserQt</class>
 <widget class="QMainWindow" name="ParserQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParserQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>400</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="paintLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
